phpstan.neon.dist

```diff
@@ -51,7 +49,9 @@ parameters:
             message: "#but it's missing from the PHPDoc @throws tag\\.$#" # allow uncatched exceptions in tests
             path: tests/*
 
-        -
-            message: '#^Call to function method_exists\(\) with PHPStan\\Analyser\\Scope and ''getKeepVoidType'' will always evaluate to true\.$#'
-            path: src/Rule/ForbidUnusedMatchResultRule.php
-            reportUnmatched: false # fails only for PHPStan > 1.10.49
+        # ignore BC promises
+        - identifier: phpstanApi.class
+        - identifier: phpstanApi.method
+        - identifier: phpstanApi.interface
+        - identifier: phpstanApi.instanceofType
```

Comment:
For example Instanceof String type can be easily fixed.

Reply:
Yeah, this one is questionable. I'll reconsider it.